### PR TITLE
Improve responsiveness for images and footer

### PR DESCRIPTION
**Pull request by georgeschena**
PURPOSE: Going through the web application and ensuring it provides a good user experience on multiple devices for images and footer.
BEFORE:

Screenshot 2019-04-29 at 16 26 43

Screenshot 2019-04-29 at 16 27 35

AFTER: 
Screenshot 2019-04-29 at 16 26 21

Screenshot 2019-04-29 at 16 26 52

### CODE REVIEW

**Comment by Renny4Real** [comment]
Could I suggest perhaps using `class: "img-fluid"` instead of `class:"image_size"`? 

`class: "img-fluid"` seems to have been used in the other support for device pages where an image was used but not on the iPhone support page _(which is the file targeted in this PR)_

I might be wrong though but would using `class: "img-fluid"` give us the desired result?
So that we don't have to add another image css class?

**Comment by georgeschena** [comment]
The image-fluid class doesn't do what I need it to do, hence the custom class I added to allow the image to not become distorted once the screen size is reduced.

File: source/stylesheets/_core.scss [hunk 1]
```diff
@@ -89,6 +89,11 @@ body {
   width: 65%;
 }
 
+.image_size {
```
Comment (Renny4Real) [inline source/stylesheets/_core.scss]: Should we rename this class, `image_size` doesn't really explain the styling.
Perhaps, `mobile_image_resize`?

Comment (georgeschena) [reply source/stylesheets/_core.scss]: I agree, changing it so its more specific

**Pull request merged**